girder-dandi-archive/girder_dandi_archive/rest.py

```diff
@@ -74,6 +75,7 @@ def create_dandiset(self, params):
             "name": name,
             "description": description,
             "identifier": padded_identifier,
```

Comment:
Yes, but I think it's out of scope for this issue. This is only to validate that schemaVersion is present and valid, we are not actually validating the schema at this time. Whenever we start doing that it will cause all these tests to fail, at which point we should change that.

Reply:
You're right, and I forgot that I actually already did that in my other branch https://github.com/dandi/dandiarchive/blob/fe9f0cd953cbac7dd0df865f78021000c7f0fe3e/girder-dandi-archive/girder_dandi_archive/rest.py#L73-L77